validator/src/main.rs

```diff
@@ -1205,10 +1205,10 @@ pub fn main() {
                 .help("Milliseconds to wait in the TPU receiver for packet coalescing."),
         )
         .arg(
```

Comment:
Retain the old `tpu-use-quic` arg please, as a `.hidden(true).conflicts_with("tpu_disable_quic")`, for a graceful update for those who are already using the old arg

Reply:
Done. Thanks for the catch